tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -58,42 +64,46 @@ class ApiGwTests
     val cliWskPropsFile = File.createTempFile("wskprops", ".tmp")
 
     /**
-      * Expected to be called before or after each CLI invocation
-      * If number of CLI invocations in this suite have reached the throttle limit
-      * then pause the test for enough time so that the throttle restriction is gone
-      */
-    def checkThrottle(maxInvocationsBeforeThrottle: Int = maxActionsPerMin) = {
-        var t = Instant.now
-        var tminus60 = t.minusSeconds(60)
-        var invocationsLast60Seconds = invocationTimes.filter(_.isAfter(tminus60)).sorted
-        var invocationCount = invocationsLast60Seconds.length
+     * Expected to be called before each CLI API command since these invoke an action.
```

Comment:
if i understand correctly, checkthrottle() is no longer called `before each CLI API command` but is now called once per test.

Reply:
Comment needs correcting then?